lib/oli_web/live/workspaces/course_author/overview_table_model.ex

```diff
@@ -0,0 +1,75 @@
+defmodule OliWeb.Workspaces.CourseAuthor.OverviewTableModel do
+  use Phoenix.Component
+
+  alias OliWeb.Common.SessionContext
+  alias OliWeb.Common.Table.{ColumnSpec, SortableTableModel}
+  alias OliWeb.Router.Helpers, as: Routes
+
+  def new(%SessionContext{} = ctx, sections) do
+    column_specs = [
+      %ColumnSpec{
+        name: :title,
+        label: "Title",
+        render_fn: &__MODULE__.custom_render/3
```

Comment:
Nitpick: Using `__MODULE__` is not necessary,  `render_fn: &custom_render/3` should work